src/modules/Comment/controller.ts

```diff
@@ -45,4 +45,34 @@ const commentOnNewsArticle = async (req: Request, res: Response) => {
   }
 };
 
-export default { commentOnNewsArticle };
+const getNewsComments = async (req: Request, res: Response) => {
+  const newsId = parseInt(req.params.newsId);
```

Comment:
As the news will originally come from outside API, we won't have a `newsId`, so here we should find the comments on NEWS based on a parameter that is coming from external API and we have it in DB as well like `URL`